packages/apps-config/src/api/spec/moonbeam.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright 2017-2020 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// structs need to be in order
+/* eslint-disable sort-keys */
+
+export default {
+  AccountId: 'EthereumAccountId',
+  Address: 'AccountId',
+  Balance: 'u128',
+  RefCount: 'u8',
```

Comment:
We actually may want to use typesBundle, i.e. versioned types. The reason is then we can do the type overrides (such as this one which will most probably change), based on a version range. (In general, since we have support for it now, it is a better option anyway, since the changes can be made even before the chain gets upgraded and will kick in on upgrade)

Reply:
Really weird. I don't want to delay this one, want to get it one if everybody is happy. (The bundles can go after)